Composer/packages/server/src/locales/en-US.json

```diff
@@ -158,6 +158,9 @@
   "add_a_trigger_c6861401": {
     "message": "Add a trigger"
   },
```

Comment:
Make sure to run `yarn l10n` before your commit. It seems like you're missing a few entries.

Reply:
Okay I ran it and updated the locale file in the PR